<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demarcate_dialog</class>
 <widget class="QDialog" name="demarcate_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>传感器标定</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/logo.jpg</normaloff>../icon/logo.jpg</iconset>
  </property>
  <widget class="QLabel" name="label_x_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>三次项</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>二次项</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>一次项</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_const">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>常数项</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_x_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_x_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_x_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_const">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_sure">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
